.env

```diff
@@ -0,0 +1 @@
+REACT_APP_SOUNDCLOUD_CLIENT_ID=da42192c93ee894169457779694b42a1
```

Comment:
Oops, don't do this! We don't want this stuff checked in. See this for more info: https://12factor.net/config

Reply:
Ok, I'll remove it. The only reason i did this was because this key was already exposed in the old player component

Reply:
Ooof 🤦‍♂️ My bad. I will remove this id, make this work properly with .env and share the token offline.

Reply:
ok sounds good!

Reply:
FYI all you'll need to do is add a .env locally and this key and node will swap it during build. And remove it from config i guess

Reply:
I used https://github.com/facebook/create-react-app/blob/master/packages/react-scripts/template/README.md#adding-custom-environment-variables

Reply:
💯